helm/alfresco-infrastructure/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-description: A Helm chart Providing the required Alfresco infrastructure for the Digital Business Platform. This includes activemq, alfresco-identity-service, akfresco-event-gateway nginx-ingress and a persistent volume to use with the DBP chart.
+description: A Helm chart Providing the required Alfresco infrastructure for the Digital Business Platform. This includes activemq, alfresco-identity-service, nginx-ingress and a persistent volume to use with the DBP chart.
 name: alfresco-infrastructure
-version: 5.1.1
+version: 5.1.1-DEPLOY-774
```

Comment:
Is the version right?

Reply:
Changing this now since i have the needed reviews.